// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/ComputedPropertyNames/parserES5ComputedPropertyName5.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,9 @@
 
 
 //// [parserES5ComputedPropertyName5.d.ts]
 interface I {
+    [e]: number;
 }
 
 /// [Errors] ////
 
